### ISSUE TITLE
Get observed state probabilities from generative process

### ISSUE THREAD

**Issue by vivekhandebagh** [Astera-org/simplexity#58]
Would be very helpful to access the vector of probabilities as a hmm produces tokens. Kind of like transformer logits but from the data's generative process instead. That would really speed up the ability do information theoretic analysis on the ground truth data.

**Comment by adamimos** [comment]
Do you mean the probabilities of the emissions or the probabiliteis over the generator states, or the probabilities of the MPS state?
